inc/template-tags.php

```diff
@@ -100,3 +100,36 @@ function _s_entry_footer() {
 	);
 }
 endif;
+
+if ( ! function_exists( '_s_post_thumbnail' ) ) :
+/**
+ * Displays an optional post thumbnail.
+ *
+ * Wraps the post thumbnail in an anchor element on index views, or a div
+ * element when on single views.
+ */
+function _s_post_thumbnail() {
+	if ( post_password_required() || is_attachment() || ! has_post_thumbnail() ) {
+		return;
+	}
+
+	if ( is_singular() ) :
+	?>
+
+	<div class="post-thumbnail">
+		<?php the_post_thumbnail(); ?>
+	</div><!-- .post-thumbnail -->
+
+	<?php else : ?>
+
+	<a class="post-thumbnail" href="<?php the_permalink(); ?>" aria-hidden="true">
+		<?php
+			the_post_thumbnail( 'post-thumbnail', array(
+				'alt' => the_title_attribute( 'echo=0' )
```

Comment:
Even though the code is GPL it still requires attribution when copying code.

The alt text should be used to the describe the image so that people with poor eyesight know what the image is. I am not convinced using the post title describes the image. https://yoast.com/image-seo-alt-tag-and-title-tag-optimization/

What I can't work out why we would have a different alt text if it is on the blog page and on a single post. I had a look at the commit history and see that the code was added by @karmatosed as part of the initial commit.